### PR TITLE
Add Korean Spelling Checker workflow

### PR DESCRIPTION
**Pull request by andromedarabbit**
This is the services workflow of Korean spelling checker for Mac OS X.

### CODE REVIEW

File: Casks/korean-spelling-checker.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  sha256 :no_check
+
+  url 'http://appletree.or.kr/automator/korean-spelling-checker-workflow.zip'
+  name '한국어 맞춤법 검사기'
+  homepage 'http://appletree.or.kr/google-chrome-extensions/'
+  license :gratis
+
+  service '✔ 선택한 글의 한국어 맞춤법 검사하기.workflow'
+
+  installer :script => '/System/Library/CoreServices/pbs',
```
Comment (jawshooah) [inline Casks/korean-spelling-checker.rb]: What does this do? From the looks of it, it belongs in a `postflight` stanza rathern than `installer :script`.

Comment (andromedarabbit) [inline Casks/korean-spelling-checker.rb]: I think you are right. I will commit a new one.

**Comment by andromedarabbit** [comment]
Could you check my new commit?

File: Casks/korean-spelling-checker.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            :args => %w[-flush],
+            :sudo => false
+
+  postflight do
+    system "/System/Library/CoreServices/pbs", "-flush"
```
Comment (vitorgalvao) [inline Casks/korean-spelling-checker.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

File: Casks/korean-spelling-checker.rb (hunk 1 above)
Comment (jawshooah) [inline Casks/korean-spelling-checker.rb]: You can delete the `installer` stanza now that you've moved the logic to `postflight`.

File: Casks/korean-spelling-checker.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  license :gratis
+
+  service '✔ 선택한 글의 한국어 맞춤법 검사하기.workflow'
+
+  installer :script => '/System/Library/CoreServices/pbs',
+            :args => %w[-flush],
+            :sudo => false
+
+  postflight do
+    system '/System/Library/CoreServices/pbs', '-flush'
```
Comment (jawshooah) [inline Casks/korean-spelling-checker.rb]: Again, what does this do? I don't see anything similar in other `service` Casks.

**Comment by andromedarabbit** [comment]
Removed `:installer` stanza and regarding `pbs`, see https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man8/pbs.8.html

        -flush
         Equivalent to -flush_pboard and -flush_userdefs, entirely resetting pbs. The next time Services
         information is needed, pbs will do a complete rescan for apps vending Services, and read their
         plist.  Note: this rescan may be very expensive!

**Comment by vitorgalvao** [comment]
Not sure this is the best way to handle this. Shouldn’t we be handling this in the implementation of `service`?

Also, there’s trailing whitespace that must be removed.

**Comment by andromedarabbit** [comment]
Trailing whitespaces? I could not find any trailing whitespaces. Did I miss things?

File: Casks/korean-spelling-checker.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  name '한국어 맞춤법 검사기'
+  homepage 'http://appletree.or.kr/google-chrome-extensions/'
+  license :gratis
+
+  service '✔ 선택한 글의 한국어 맞춤법 검사하기.workflow'
+
+  postflight do
+    system '/System/Library/CoreServices/pbs', '-flush'
+  end
+
```
Comment (jawshooah) [inline Casks/korean-spelling-checker.rb]: This is the line with trailing whitespace according to [Travis](https://travis-ci.org/caskroom/homebrew-cask/jobs/70394252#L2240).

**Comment by andromedarabbit** [comment]
Thanks for helping me. Finally I realized that I have missed audit process.

File: Casks/korean-spelling-checker.rb (hunk 3 above)
Comment (vitorgalvao) [inline Casks/korean-spelling-checker.rb]: This cask won’t be merged until @jawshooah’s question is made clear.

Comment (andromedarabbit) [inline Casks/korean-spelling-checker.rb]: Removed :installer stanza and regarding pbs, see https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man8/pbs.8.html

>-flush
     Equivalent to -flush_pboard and -flush_userdefs, entirely resetting pbs. The next time Services
     information is needed, pbs will do a complete rescan for apps vending Services, and read their
     plist.  Note: this rescan may be very expensive!

Comment (andromedarabbit) [inline Casks/korean-spelling-checker.rb]: Running 'pbs -flush' reveals the new service in the menu.

**Comment by andromedarabbit** [comment]
Is there anything I can do to make this to be merged?

**Comment by fanquake** [comment]
@andromedarabbit Can you please squash your commits

**Comment by vitorgalvao** [comment]
Remove the postflights, please. Those need to be discussed in a different issue.

**Comment by andromedarabbit** [comment]
I removed posflights and made a new pull request since I have accidently deleted my old repository.

**Pull request closed without merging**